web/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
Why not an es6 or even newer one?

Reply:
For instance, if you want to target IE11, you'll have to target ES5. If you want to support NodeJS, Edge or Chromium, ES6 is ok.
I think it's good practice to support older versions of a browser, so let's stick with es5 for now.

Reply:
This was my misunderstanding. Agree with you